lib/ace/schemas/ace-run_task.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "ACE run_task request",
+  "description": "POST /run_task request schema for ACE",
+  "type": "object",
+  "properties": {
+    "target": {
+      "type": "object",
+      "description": "Contains the Transport schema to connect to the remote target",
+      "properties": {
+        "remote-transport": {
+          "type": "string",
+          "description": "The name of the transport being used"
+        },
+        "run-on": {
+          "type": "string",
+          "description": ""
```

Comment:
@adreyer I need some feedback on what the description of this should be. Not required for this PR. This came from notes I took during a call with you and @DavidS